extension/httpfs/include/s3fs.hpp

```diff
@@ -73,19 +73,21 @@ class S3FileHandle : public HTTPFileHandle {
 	friend class S3FileSystem;
 
 public:
-	S3FileHandle(FileSystem &fs, std::string path_p, uint8_t flags, const HTTPParams &http_params,
-	             const S3AuthParams &auth_params_p, const S3ConfigParams &config_params_p)
+	S3FileHandle(FileSystem &fs, std::string path_p, std::string &stripped_path_p, uint8_t flags,
```

Comment:
Same as before `std::` can be removed here as well

Reply:
✔